cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:3001",
+  "testFiles": ["**/*.spec.ts"]
```

Comment:
You might want to add a configuration to retry the test if it fails 💡 

```suggestion
  "testFiles": ["**/*.spec.ts"],
  "retries": {
      "runMode": 3,
      "openMode": 2
    }
```

Reply:
Added retry mode https://github.com/Lahphim/react-surveyor-challenge/pull/47/commits/3d79d932966ae768dbf9450c2a220af575052d36